release-plz.toml

```diff
@@ -50,9 +50,7 @@ changelog_path = "./CHANGELOG.md"
 name = "c2patool"
 
 [[package]]
-name = "c_api"
-publish = false
-release = false
```

Comment:
I'm not sure we want to release this, since that will create another gitbhub release, I think. I don't really know release-plz either. We may want to check with Eric on this.

Reply:
To be prepared, let me make those changes in another branch. Worse case we can pick the one over the other.

Reply:
Update: Looking at the renaming, the mod name comes from the file. I would keep `c_api(.rs)` in the folder, even if we decide to rename the folder. That's the change done in https://github.com/contentauth/c2pa-rs/pull/1159/. I flagged it for the workflows to run to verify if that one can build too.